### ISSUE TITLE
key-spacing.align doesn't like line-breaks

### ISSUE THREAD

**Issue by zyklus** [eslint/eslint#3125]
This works fine:

```js
let x = {
  foo    : 42,
  foobar : 43
}
```

This doesn't:
```js
let x = {
  foo    : 42,

  foobar : 43
}
```

throws `key-spacing: Extra space after key "foo".`

Also, comments should work:

```js
let x = {
  foo    : 42,

  // foobar comment
  foobar : 43
}
```

eslint version = `v0.24.1`

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by zyklus** [comment]
Also, I fully realize that there are multiple use-cases here.  I think either should work:

```js
key        : val
anotherKey : val

foo        : val
bar        : val
```

```js
key        : val,
anotherKey : val,

foo : val,
bar : val,
```

**Comment by IanVS** [comment]
The reason your second case is failing is that the group after the empty line is seen as a new group (as discussed in https://github.com/eslint/eslint/issues/1439).  

What is it you're asking for here?

**Comment by nzakas** [comment]
@btmills

**Comment by btmills** [comment]
@zyklus are you asking for what's proposed in #2668?

**Comment by zyklus** [comment]
@btmills, yes that's basically what I'm asking for.  I'll chime in there.

**Comment by btmills** [comment]
Closing in favor of #2668.
